src/blueapi/worker/worker.py

```diff
@@ -11,6 +12,13 @@
 T = TypeVar("T")
 
 
+class TaskStatusEnum(str, Enum):
```

Comment:
Nit: I don't like putting what it is in the name

```suggestion
class TaskState(str, Enum):
```

Reply:
pls noooooo, 
that'd be a 3rd form of task state. we need this for differentiation, interim while the refactor is not here.

Reply:
Happy with that as long as there is a comment or issue saying it is interim